pkg/dagger.io/dagger/core/fs.cue

```diff
@@ -72,10 +72,12 @@ import "dagger.io/dagger"
 	input: dagger.#FS
 	// Contents to copy
 	contents: dagger.#FS
+	// Working dir (optional)
+	workdir: string | *"/"
```

Comment:
Saw your comment about which OS to use for operations in the `path` package. I think we can safely assume `path.Unix`. Even though we support Dagger clients running on many platforms, AFAIK all dagger actions can only be expected to run with unix filepaths.

I think that would be a better approach overall too by keeping our `core` interface as minimal as possible; seems like it should allow the change exist only in `docker/build.cue`.

Reply:
Awesome, that's exactly my follow up question. I'll rework it and resubmit with that.